cluster/k8s1.6/bootstrap_centos.sh

```diff
@@ -18,7 +18,7 @@ EOF
 
 setenforce 0
 
-yum install -y docker kubelet-1.6.5 kubeadm-1.6.5 kubectl-1.6.5 kubernetes-cni-1.6.5 ntp
+yum install -y docker kubelet-1.6.5 kubeadm-1.6.5 kubectl-1.6.5 kubernetes-cni ntp
```

Comment:
We don't need to explicitly pick a version any more? Seems like we'd want to limit this.